### PR TITLE
whiteboard complete

### PR DESCRIPTION
**Pull request by brandon-fenty**

### CODE REVIEW

**Comment by brandon-fenty** [comment]
-[x] Write a method that takes a number (k) and returns the node at that index in a linked list
-[x] Write tests that prove functionality of the method accounting for hedge cases
-[x] Document the challenge and solution into a README file

File: src/day06/LinkedList.java [hunk 1]
```diff
@@ -122,4 +122,23 @@ public void printList () {
         // Last node
         System.out.print(current);
     }
+
+    public ListNode kthElement (int k) {
+        int elCount = 0;
+        ListNode tempNode = this.root;
+
```
Comment (mollyfish) [inline src/day06/LinkedList.java]: Great comments

File: src/day07/KthElementTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import day06.LinkedList;
+import day06.ListNode;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class KthElementTest {
+    LinkedList n1 = new LinkedList();
+
```
Comment (mollyfish) [inline src/day07/KthElementTest.java]: Woohoo for `@BeforeEach`!  Isn't that an awesome time saver?

File: src/day07/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Kth from the end of a linked list
+
+- Given a linked list
+- Methods created in the LinkedList class can be used.
+
+## Challenge
+
+- Write a function for the linked list class that will accept a number (k) as a parameter and return the node from the index of k (the index starts at the end of the list).
+
```
Comment (mollyfish) [inline src/day07/README.md]: Does the index start counting at `0` like an array or `1` like normal numbers? This is an important piece of info to offer your users.
